systemtests/src/main/java/io/enmasse/systemtest/TestUtils.java

```diff
@@ -87,7 +87,7 @@ public static void waitForNReplicas(Kubernetes kubernetes, int expectedReplicas,
     public static void waitForNBrokerReplicas(Kubernetes kubernetes, String infraUuid, int expectedReplicas, boolean readyRequired,
                                               Destination destination, TimeoutBudget budget, long checkInterval) throws InterruptedException {
         String statefulsetName = destination.getDeployment().startsWith("broker") ?
-                String.format("broker-pooled-%s", infraUuid) : destination.getDeployment();
+                String.format("broker-%s-0", infraUuid) : destination.getDeployment();
```

Comment:
why is there suffix "-0" ?

Reply:
Fixed. It now resolves the actual cluster id of the destination instead